setup.py

```diff
@@ -37,7 +37,7 @@
     'dill>=0.2.5, <0.3dev',
     'future>=0.16.0, <0.17dev',
     'googleapis-common-protos>=1.5.2, <2.0dev',
-    'grpcio>=1.0.0, <2.0dev',
+    'grpcio>=1.1.3, <2.0dev',
```

Comment:
Any idea where that will cause problems in the real world outside of CI? (My guess is not,  but I was unsure. I guess if the user _already has_ grpcio installed, obviously it is not a problem...)